deploy-manage/cloud-organization/billing/elastic-observability-billing-dimensions.md

```diff
@@ -22,6 +25,10 @@ Data volumes for ingest and retention are based on the fully enriched normalized
 
 ## Synthetics [synthetics-billing]
 
-[Synthetic monitoring](/solutions/observability/synthetics/index.md) is an optional add-on to Observability Serverless projects that allows you to periodically check the status of your services and applications. In addition to the core ingest and retention dimensions, there is a charge to execute synthetic monitors on our testing infrastructure. Browser (journey) based tests are charged per-test-run, and ping (lightweight) tests have an all-you-can-use model per location used.
+[Synthetic monitoring](/solutions/observability/synthetics/index.md) is an optional add-on to Observability Serverless projects that allows you to periodically check the status of your services and applications as a part of your "Complete" tier subscription. In addition to the core ingest and retention dimensions, there is a charge to execute synthetic monitors on our testing infrastructure. Browser (journey) based tests are charged per-test-run, and ping (lightweight) tests have an all-you-can-use model per location used.
```

Comment:
wonder if this would be clearer if we put the tiers up above, like 

Tier: Complete
Opt-in only

<paragraph>

but won't make you refactor for my sake